build.gradle.kts

```diff
@@ -215,14 +215,18 @@ tasks {
 
 	compileJava {
 		options.encoding = "UTF-8"
-		options.compilerArgs.add("-Werror")
-		options.compilerArgs.add("-Xlint:all")
+		if (!modularBuild.toBoolean()) {
+			options.compilerArgs.add("-Werror")
+			options.compilerArgs.add("-Xlint:all")
+		}
 	}
 
 	compileTestJava {
 		options.encoding = "UTF-8"
-		options.compilerArgs.add("-Werror")
-		options.compilerArgs.add("-Xlint:all")
+		if (!modularBuild.toBoolean()) {
```

Comment:
This is probably not necessary. The test sources don't seem to cause any problems for modular builds.